package.json

```diff
@@ -15,7 +15,7 @@
     "shrink": "npmshrink",
     "start": "node ./bin/db_patcher.js >/dev/null && node ./bin/server.js",
     "start-mem": "node ./bin/mem",
-    "test": "npm run test-mysql && npm run test-mem && npm run test-server",
+    "test": "npm run test-mysql && npm run test-mem && npm run test-server && grunt eslint",
```

Comment:
Got caught by this, in auth-server we run linting after completing tests. Updated this to do the same.